src/app/feature/map/search/search.component.ts

```diff
@@ -16,9 +22,21 @@ export class SearchComponent implements OnInit {
   constructor(
     private readonly _mapService: MapService,
     private readonly _mapDataProviderService: MapDataProviderService,
+    private readonly _mapPathService: MapPathService,
     private readonly _translateService: TranslateService
   ) {}
 
+  public readonly selectedUserLocation$ = this._mapPathService.startPoint$.pipe(
+    filter((point) => !isNil(point)),
```

Comment:
Please use positive value instead  `filter((point) => !isNil(point)),`
And check other code